tests/test/export/test.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+function verify_output
+{
+    typeset output_type=${1?"*** yaml OR dict"}
+    shift
+    typeset fields="$*"
+
+    typeset field=""
+    for field in $fields; do
+        if [[ "$output_type" == "yaml" ]]; then
+            rlAssertGrep "$field:"   $rlRun_LOG
+        else
+            rlAssertGrep "'$field':" $rlRun_LOG
+        fi
+    done
+}
+
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "pushd data"
+        rlRun "set -o pipefail"
+    rlPhaseEnd
+
+    tnames=""
+    tnames+=" /tests/test/export/data/tests/enabled/default"
+    tnames+=" /tests/test/export/data/tests/enabled/defined"
+    tnames+=" /tests/test/export/data/tests/enabled/disabled"
+    tnames+=" /tests/test/export/data/tests/tag/default"
+    tnames+=" /tests/test/export/data/tests/tag/defined"
+    tnames+=" /tests/test/export/data/tests/tag/empty"
+    tnames+=" /tests/test/export/data/tests/tier/default"
+    tnames+=" /tests/test/export/data/tests/tier/one"
+    tnames+=" /tests/test/export/data/tests/tier/two"
```

Comment:
Please, do a `tmt init` in the `/tests/test/export/data` directory to mark root of a new `fmf.Tree`. The `data` directory should include its own `.fmf` directory so that the export test data do not overlap/mix with the main fmf metadata.

Reply:
Done.